policies/bundles/TransportEncryptionAlgorithm/input.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This is for testing, right? I'm thinking about putting them in a dedicated test folder. What do you think?